srcpkgs/base-files/INSTALL

```diff
@@ -48,6 +48,10 @@ make_system_dirs() {
 		install -d usr/lib32
 		ln -sf usr/lib32 lib32
 	fi
+	if [ "${ARCH}" = "ppc64le" ]; then
+		ln -sf usr/lib lib64
+		cd usr && ln -sf lib lib64 && cd ..
```

Comment:
Why can't you avoid the chdir here?

Reply:
No particular reason. I just copied this bit from the x86_64 condition above.